strawberry/auto.py

```diff
@@ -57,7 +57,7 @@ def __instancecheck__(
             if args[0] is Any:
                 return any(isinstance(arg, StrawberryAuto) for arg in args[1:])
 
-        return False
+        return instance == "strawberry.auto"
```

Comment:
:D

Reply:
yeah it complicated because we only resolve the annotations at a later point of time......

Reply:
I wonder if we can leverage what we do for strawberry.type, but probably not since the type resolution is deferred quite a bit 🤔

Reply:
@patrick91 think we should merge this in the meantime XD